SETUP_SUBMITTER_CMF.md

```diff
@@ -86,6 +86,41 @@ Note that you may wish to disable the "strict hash" feature in Unreal's Python s
 
 3.  If you're pulling from mainline you may have python dependencies which are not yet released to PyPi - you'll need to build and install your local copy which can be done with hatch.  Note that the .whl file will need to be changed to reflect the version which is output by hatch build:
 
+```
+// Install hatch if not yet installed
+pip install hatch
+hatch build
+"C:\Program Files\Epic Games\UE_5.4\Engine\Binaries\ThirdParty\Python3\Win64\python" -m pip install dist\deadline_cloud_for_unreal_engine-0.2.2.post21-py3-none-any.whl --target "C:\Program Files\Epic Games\UE_5.4\Engine\Plugins\UnrealDeadlineCloudService\Content\Python\libraries"
+```
+
+4.  Lastly, Python dependencies can be installed by the submitter installer.  NOTE - these may be out of date with your code above from the release or mainline branch, and this method should not currently be preferred.
+
+1. If you've built and installed the plugin from the release branch above, you can simply install from pip. Use the following install command, adjusting the paths to your Unreal installation:
```

Comment:
The number looks a little funny here in markdown, and it renders with different numbers. Does the list need to be sorted out?

Reply:
Looks like a bad merge, thanks!